packages/test-utils/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@fuel-wallet/test-utils",
```

Comment:
We can add this as private for now, and release once we achieve a stable version